### ISSUE TITLE
Prevent multiple log-ins

### ISSUE THREAD

**Issue by teravest** [qiime/qiime_web_app#540]
We need to prevent multiple logs-ins from the same browser

**Comment by teravest** [comment]
The real issue here is not being able to overwrite a study.  I just thought of another way to fix this.  
The real issue here is on edit study the study_id is coming from sess when the submit button is pressed.  What if when the page is loaded we save the study id, then we have two options.  We can submit the saved study id or we can verify the study id is the same as what is in sess and only submit if that is true. 

Any thoughts?

**Comment by adamrp** [comment]
Yeah, i think that could work!

**Comment by ElDeveloper** [comment]
That sounds good, would we also save the study identifier in the session dictionary?

:+1:

**Comment by adamrp** [comment]
Yes, I think the study_id needs to be stored in the session (as it
currently is), and if we store it also in a hidden form input as Emily
suggests, we should be able to use that (instead of the study_id in the
session) when updating study information.  The reason to keep it stored in
the session as well is that there may be other parts of the web app that
expect it to be there and depend on it.

**Comment by teravest** [comment]
I am not changing anything that already exists in the session.  Just adding
it to the form and using the form value instead of the session value when
editing.
As soon as I fix my git failure I will issue a pull request.
